framework/src/tinyalsa/tinyalsa.c

```diff
@@ -739,6 +779,10 @@ int pcm_close(struct pcm *pcm)
 	if (pcm->fd >= 0) {
 		close(pcm->fd);
 	}
+
+	pcm->nextSize = 0;
+        pcm->nextOffset = 0;
```

Comment:
Please align code here.

Reply:
align is not enough, you should use tab as an indentation. (you used spaces)

Reply:
Fixed